### ISSUE TITLE
Command on level not working

### ISSUE THREAD

**Issue by Endothelium** [Zrips/Jobs#350]
Hello,

I cant use this feature 

```
commands:
    kit:
      command:
      - lp user [player] permission add wtk.job.bucheron10
      - msg [player] !Permission bucheron10 
      levelFrom: 1
      levelUntil: 10
```

When i level from 1 to 2, the commands are not performed.

I am on spigot 1.13.2

**Comment by montlikadani** [comment]
This works properly, only you have a configuration error because it has more space than it needed.
The correct:
```
commands:
  kit:
    command:
    - lp user [player] permission add wtk.job.bucheron10
    - msg [player] !Permission bucheron10 
    levelFrom: 1
    levelUntil: 10
```

**Comment by Endothelium** [comment]
There is an indentation problem when i pasted my config here, but actually the indentation is good in my file and is exactely the same as  you.

**Comment by montlikadani** [comment]
This working for me correctly when I reach the 2 level from 1 and gets the commands. Make sure that the commands you set are working even if you type them in the game.

**Comment by Endothelium** [comment]
Hello,

Yeah sorry, it does work, it was because i was using alias command instead the real command.

Another question, does this seems correct for you, except indentation

```
permissions:
    wtk.job.bucheron:
      value: true
      level: 0
    wtk.job.bucheron10:
      value: true
      level: 10
```

Because i have trouble to make this work too.

Thanks

**Comment by montlikadani** [comment]
Unfortunately, the permission and conditions section has some minor permission setup problems. The problem is that it does not give the permission to the permission system. Same as #217

**Comment by DonSensiLeone** [comment]
excuse me if i'm joining the conversation (just let me know if i need to open a new issue..) but i do encounter the same problem that was firstly stated here (more or less) , i can get the reward+ message for level 5 , nothing at all for level 10, 15,20,25 are working correctly , then level 30 doesn't give anything... 
[jobexample.txt](https://github.com/Zrips/Jobs/files/2898490/jobexample.txt)

**Comment by DonSensiLeone** [comment]
Bucheron:
    fullname: Bûcheron
    shortname: B
    description: Gagne des sous en plantant et coupant des arbres
    ChatColour: DARK_GREEN
    chat-display: shortjob
    max-level: 200
    leveling-progression-equation: 10*(joblevel)+(joblevel*joblevel*4)
    income-progression-equation: baseincome+(baseincome*(joblevel-1)*0.01)-((baseincome+(joblevel-1)*0.01) * ((numjobs-1)*0.05))
    points-progression-equation: basepoints+(basepoints*(joblevel-1)*0.01)-((basepoints+(joblevel-1)*0.01) * ((numjobs-1)*0.05))
    experience-progression-equation: baseexperience-(baseexperience*((numjobs-1) *0.01))
    rejoinCooldown: 10
    Gui:
      Id: 271
      Data: 0
    commands:
      # command name, just to have better idea what this do
      joinconstructeur:
        # Command its self, this will be executed from console, so all commands should work
        # Possible variables are: [player] [jobname] [oldlevel] [newlevel]
        command: lp user [player] permission set jobs.join.constructeur true
        # When to execute this command first time
        levelFrom: 80
        # Until when to do this command
        # This can be set to same level as levelFrom, so this command will be executed only once
        levelUntil: 80
      message:
        command: msg [player] Vous avez débloqué le métier de Constructeur !
        levelFrom: 80
        levelUntil: 80
      level5:
        command:
        - eco give [player] 20
        - msg [player] Paie pour avoir atteint le niveau 5 de votre job !
        levelFrom: 5
        levelUntil: 5
      level10:
        command:
        - eco give [player] 50
        - msg [player] Paie pour avoir atteint le niveau 10 de votre job !
        levelFrom: 10
        levelUntil: 10
      level15:
        command:
        - eco give [player] 90
        - msg [player] Paie pour avoir atteint le niveau 15 de votre job !
        levelFrom: 15
        levelUntil: 15
      level20:
        command:
        - eco give [player] 140
        - msg [player] Paie pour avoir atteint le niveau 20 de votre job !
        levelFrom: 20
        levelUntil: 20
      level25:
        command:
        - eco give [player] 200
        - msg [player] Paie pour avoir atteint le niveau 25 de votre job !
        levelFrom: 25
        levelUntil: 25
      level30:
        command:
        - eco give [player] 270
        - msg [player] Paie pour avoir atteint le niveau 30 de votre job !
        levelFrom: 30
        levelUntil: 30
      level40:

**Comment by montlikadani** [comment]
@DonSensiLeone I checked this, and this working correctly for me when I reach the 30 level. Do you send me the full job that this problem occurs?

**Comment by DonSensiLeone** [comment]
No i didn't send you the whole job but it's the same things until level 200 , i'm not sure that you understood correctly so i'm going to try to explain it again : when me or players from my servers are joining this particular job , we can get the message + eco command for leveling to level 5 , then once we reach level 10 , nothing happens at all (no message and no eco command) , then we can reach each levels (15,20,25) and get the message and the eco command but then , as we reach level 30 , its the same than reaching level 5 , no message and no eco command .. and i get this 'error' as well on my test server as on my official server , maybe is it related to essentials ? i don't know.. i'm kinda lost there as i seem to not messing up my job setup (or maybe do i ?)  Here is a paste of the full job : https://pastebin.com/fhEDLcEn

**Comment by DonSensiLeone** [comment]
Since this last message i bought CMI , replaced essentials and other plugins with this.. and tested the job and i still get the same leveling errors at the exact same darn levels :/ i made the change in my job config for **cmi money give [player] amount**

**Comment by montlikadani** [comment]
> 
> 
> Hello,
> 
> Yeah sorry, it does work, it was because i was using alias command instead the real command.
> 
@DonSensiLeone 
Check this message.

**Comment by DonSensiLeone** [comment]
This error happens too on other jobs (same config than this one) ,as Miner , same levels are impacted with commands not working correctly.

**Comment by Zrips** [comment]
@DonSensiLeone 
You have duplicated names like
`      level10:
        command:
        - eco give [player] 50
        - msg [player] Paie pour avoir atteint le niveau 10 de votre job !
        levelFrom: 10
        levelUntil: 10`
and
`      level10:
        command:
        - eco give [player] 1000
        - msg [player] Paie pour avoir atteint le niveau 110 de votre job !
        levelFrom: 110
        levelUntil: 110`

**Comment by DonSensiLeone** [comment]
Ohh ! that would explain ! i try to change and keep in touch
